<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>200</height>
    </size>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_note"/>
    <addaction name="actionOuvrir_une_note"/>
    <addaction name="actionNote_r_cente"/>
    <addaction name="separator"/>
    <addaction name="actionEnregistrer_la_note"/>
    <addaction name="actionEnrergister_sous"/>
    <addaction name="separator"/>
    <addaction name="actionFermer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionR_tablir"/>
    <addaction name="actionCouper"/>
    <addaction name="actionCopier"/>
   </widget>
   <widget class="QMenu" name="menuOutils">
    <property name="title">
     <string>Outils</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFen_tre">
    <property name="title">
     <string>Fenêtre</string>
    </property>
    <addaction name="actionPlein_cran"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuOutils"/>
   <addaction name="menuFen_tre"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTexte"/>
  </widget>
  <action name="actionNouvelle_note">
   <property name="icon">
    <iconset>
     <normaloff>../build-Note-Desktop_Qt_5_3_MinGW_32bit-Debug/debug/icon/text.png</normaloff>../build-Note-Desktop_Qt_5_3_MinGW_32bit-Debug/debug/icon/text.png</iconset>
   </property>
   <property name="text">
    <string>Nouvelle note</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+N</string>
   </property>
  </action>
  <action name="actionOuvrir_une_note">
   <property name="text">
    <string>Ouvrir une note</string>
   </property>
  </action>
  <action name="actionNote_r_cente">
   <property name="text">
    <string>Note récente</string>
   </property>
  </action>
  <action name="actionEnregistrer_la_note">
   <property name="text">
    <string>Enregistrer </string>
   </property>
  </action>
  <action name="actionEnrergister_sous">
   <property name="text">
    <string>Enrergister sous</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset>
     <normaloff>../build-Note-Desktop_Qt_5_3_MinGW_32bit-Debug/debug/icon/block.png</normaloff>../build-Note-Desktop_Qt_5_3_MinGW_32bit-Debug/debug/icon/block.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionR_tablir">
   <property name="text">
    <string>Rétablir</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="text">
    <string>Couper</string>
   </property>
  </action>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionPlein_cran">
   <property name="text">
    <string>Plein écran</string>
   </property>
  </action>
  <action name="actionTexte">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../build-Note-Desktop_Qt_5_3_MinGW_32bit-Debug/debug/icon/pen.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Texte</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
